ibazel/fs_watcher_darwin.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2017 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build darwin
+
+package main
+
+import (
+	"os"
+
+	"github.com/bazelbuild/bazel-watcher/ibazel/fs_watcher"
```

Comment:
What would you think about making this a part of `github.com/bazelbuild/bazel-watcher/ibazel/fswatcher` and then moving 

```
github.com/bazelbuild/bazel-watcher/ibazel/fsnotify_watcher => github.com/bazelbuild/bazel-watcher/ibazel/fswatcher/fsnotify_watcher
github.com/bazelbuild/bazel-watcher/ibazel/fsevents_watcher => github.com/bazelbuild/bazel-watcher/ibazel/fswatcher/fsevents_watcher